app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,54 @@
+class InvitesController < ApplicationController
+  skip_before_action :authenticate!, only: :accept
+
+  def create
+    begin
+      unless Invite.where('email = ? AND website_id = ? AND invited_by_id =?', invite_params[:email], current_website.id, current_user.id).blank?
+        raise "Duplicate invites"
+      end
+      invite = current_website.invites.new(invited_by_id: current_user.id, email: invite_params[:email])
+      UserMailer.member_invitation(invite.id).deliver_later if invite.save
+      redirect_to new_invite_url, notice: 'Email sent'
+
+    rescue Exception => e
+      redirect_to(:action => 'new', notice: e.message)
+    end
+  end
```

Comment:
I didn't prefered to use return since last 2 times when i wanted to return false but the execusion still continued.. maybe it was just a bad experience